bitnami/spark/values.yaml

```
@@ -286,6 +286,11 @@ master:
   ## @param master.resources.limits The resources limits for the container
   ## @param master.resources.requests The requested resources for the container
   ##
+  ## NFS
+  ## @param master.nfs_server_ip Optionally specify the IP of NFS Server
+  ## @param master.nfs_server_path: Optionally specify the NFS server volume path (required if nfs_server_ip is defined)
+  ## @param master.nfs_mount_path: Optionally specify the local path that will mount the NFS share volume (required if nfs_server_ip is defined)
+
```

You need to include the parameter under the comment too:

```suggestion
  ## NFS
  ## @param master.nfs_server_ip Optionally specify the IP of NFS Server
  nfs_server_ip: ""
  ## @param master.nfs_server_path: Optionally specify the NFS server volume path (required if nfs_server_ip is defined)
  nfs_server_path: ""
  ## @param master.nfs_mount_path: Optionally specify the local path that will mount the NFS share volume (required if nfs_server_ip is defined)
  nfs_mount_path: ""
```

You could use line 280, under ` extraVolumeMounts: []`